prow/plugins/cat/cat.go

```diff
@@ -137,6 +138,9 @@ func (c *realClowder) URL(category string, movieCat bool) string {
 }
 
 func (c *realClowder) readCat(category string, movieCat bool) (string, error) {
+	if category == "no" {
```

Comment:
nit: You could use a regular expression on the category to also match `category` values like "nope" and "grumpy".

Reply:
I don't know regex well so I'll hold off on this